index.html

```diff
@@ -7,50 +7,48 @@
     <title>Commonplace Cooperative</title>
     <link rel="stylesheet" href="style.css">
   </head>
-  
-  <header>
-    <!-- Header section -->
-    <div class="header">
-      <!-- Nav Bar -->
-        <div class="nav-bar">
-           <div class="left-side"> <!--rename elements to <nav> ?-->
-            <img id="logo" src="/images/logo.png" alt="commonplace-logo">
-            <p id="commonplace-coop">commonplace.coop</p>
+  <body>
+    <header>
```

Comment:
Even though a `header` element is meant for the top of the page, it still belongs inside the `body`. `body` is part of the HTML spec and is where all your rendered elements should go.

Reply:
Ahhh right!